src/js/components/Ballot/OfficeItemCompressed.jsx

```diff
@@ -388,56 +391,56 @@ class OfficeItemCompressed extends Component {
               if (!oneCandidate || !oneCandidate.we_vote_id) {
                 return null;
               }
 
-              const voterSupportsThisCandidate = SupportStore.get(oneCandidate.we_vote_id) && SupportStore.get(oneCandidate.we_vote_id).is_support;
+              // const voterSupportsThisCandidate = SupportStore.get(oneCandidate.we_vote_id) && SupportStore.get(oneCandidate.we_vote_id).is_support;
               const candidatePartyText = oneCandidate.party && oneCandidate.party.length ? `${oneCandidate.party}` : '';
 
               // If at the end the candidates, none were chosen by voter, we use this preview list.
               candidatePreviewCount += 1;
               if (candidatePreviewCount <= candidatePreviewLimit) {
                 oneCandidateDisplay = (
-                  <div key={`candidate_preview-${oneCandidate.we_vote_id}`} className="u-stack--md u-gray-border-bottom">
-                    <CandidateInfo>
+                  <CandidateInfo
+                    key={`candidate_preview-${oneCandidate.we_vote_id}`}
+                    brandBlue={theme.palette.primary.main}
+                    onClick={() => this.goToCandidateLink(oneCandidate.we_vote_id)}
+                  >
+                    <CandidateTopRow>
                       {/* Candidate Image */}
                       <Candidate>
-                        <Link to={this.getCandidateLink(oneCandidate.we_vote_id)}>
-                          <ImageHandler
-                            className="card-main__avatar-compressed"
-                            sizeClassName="icon-candidate-small u-push--sm "
-                            imageUrl={oneCandidate.candidate_photo_url_large}
-                            alt="candidate-photo"
-                            kind_of_ballot_item="CANDIDATE"
-                          />
-                        </Link>
+                        <ImageHandler
+                          className="card-main__avatar-compressed"
+                          sizeClassName="icon-candidate-small u-push--sm "
+                          imageUrl={oneCandidate.candidate_photo_url_large}
+                          alt="candidate-photo"
+                          kind_of_ballot_item="CANDIDATE"
+                        />
                         {/* Candidate Name */}
                         <div>
-                          <Link to={this.getCandidateLink(oneCandidate.we_vote_id)}>
-                            <h4 className="card-main__candidate-name card-main__candidate-name-link u-f5">
-                              {oneCandidate.ballot_item_display_name}
-                              <br />
-                              <span className="card-main__candidate-party-description">{candidatePartyText}</span>
-                            </h4>
-                          </Link>
+                          <h4 className="card-main__candidate-name card-main__candidate-name-link u-f5">
```

Comment:
Hi Leon, I haven't tested this code yet, but I believe we still need the Links in place here.